prow/test/integration/setup-cluster.sh

```diff
@@ -109,7 +109,7 @@ function deploy_prow() {
   do-kubectl delete -A ValidatingWebhookConfiguration ingress-nginx-admission
   do-kubectl create configmap config --from-file=config.yaml=${CONFIG_ROOT_DIR}/config.yaml --dry-run=client -oyaml | kubectl apply -f -
   do-kubectl create configmap plugins --from-file=plugins.yaml=${CONFIG_ROOT_DIR}/plugins.yaml --dry-run=client -oyaml | kubectl apply -f -
-  do-kubectl create configmap job-config --from-file=periodics.yaml=${CONFIG_ROOT_DIR}/jobs/periodics.yaml --dry-run=client -oyaml | kubectl apply -f -
+  do-kubectl create configmap job-config --from-file=${CONFIG_ROOT_DIR}/jobs/ --dry-run=client -oyaml | kubectl apply -f -
```

Comment:
I didn't know this works, could you paste how the yaml looks like? basically the output of:

```
kubectl create configmap job-config --from-file=${CONFIG_ROOT_DIR}/jobs/ --dry-run=client -oyaml 
```

Reply:
```
apiVersion: v1
data:
  periodics.yaml: |
    periodics:
    - interval: 1m
      name: horologium-schedule-test-job
      spec:
        containers:
        - command:
          - echo
          args:
          - "Hello World!"
          image: localhost:5001/alpine
  presubmits.yaml: |
    presubmits:
      http://fakegerritserver/gerrit-test-infra:
        - name: hello-world-presubmit
          always_run: true
          labels:
            preset-dind-enabled: "true"
            preset-kind-volume-mounts: "true"
            preset-cookie-file: "true"
          spec:
            containers:
            - command:
              - echo
              args:
              - "Hello World!"
              image: localhost:5001/alpine
kind: ConfigMap
metadata:
  creationTimestamp: null
  name: job-config
 ```